modules/client-configuration/pages/autoinst-profiles.adoc

```diff
@@ -1,36 +1,45 @@
 
-You can create your autoinstallation profile directly in the {productname} {webui}.
-The simplest way to create a profile is to upload an {ay} or {kickstart} file.
-Alternatively, for {kickstart} only, you can use the {webui} wizard.
+See xref:client-configuration:autoinstallation-profiles.adoc#autoyast[Next section] for generic instructions on how to declare a profile.
 
-Before you begin, you must have created an autoinstallation distribution, and prepared an autoinstallation file.
-For more information about autoinstallation distributions and files, see xref:client-configuration:autoinst-setup.adoc[].
+See xref:client-configuration:autoinstallation-profiles.adoc#autoyast[AutoYast Profiles] for pecularities of the {ay} profiles.
 
+See xref:client-configuration:autoinstallation-profiles.adoc#kickstart[Kickstart Profiles] for pecularities of the {kickstart} profiles.
 
+The autoinstallation file contained in the profile may use variables and code snippets. For details, see xref:client-configuration:autoinstallation-profiles.adoc#templates-syntax[Templates Syntax].
+
+
+[[declare-profile]]
+== Declare the Profile
+
+When you have prepared an autoinstallation file and distribution, you can create profiles to manage autoinstallation on your {productname} Server. For a given distribution, the profile will determine how to install this distribution.
+
+You can create your autoinstallation profile directly in the {productname} {webui}. To see and manage your autoinstallation profiles, navigate to menu:Systems[Autoinstallation > Profiles].
+
+One way to create a profile is to upload an {ay} or {kickstart} file. Alternatively, for {kickstart} only, you can use the {webui} wizard.
 
 .Procedure: Creating an Autoinstallation Profile by Upload
 . In the {productname} {webui}, navigate to menu:Systems[Autoinstallation > Profiles].
 . Click btn:[Upload Kickstart/Autoyast File].
-. In the [guimenu]``Label`` field, type a name for the profile.
-    Do not use spaces.
+. In the [guimenu]``Label`` field, type a name for the profile. Do not use spaces.
 . In the [guimenu]``Autoinstall Tree`` field, select the autoinstallation distribution to use for this profile.
 . In the [guimenu]``Virtualization Type`` field, select the type of virtualization to use for this profile, or select ``None`` for no virtualization.
-. Copy the contents of your autoinstallation file into the [guimenu]``File Contents`` field, or upload the file directly using the [guimenu]``File to Upload`` field.
+. Copy the contents of your autoinstallation file into the [guimenu]``File Contents`` field, or upload the file directly using the [guimenu]``File to Upload`` field. Refer
+to section xref:client-configuration:autoinstallation-profiles.adoc#autoyast[AutoYast Profiles] or xref:client-configuration:autoinstallation-profiles.adoc#kickstart[Kickstart Profiles] for information about what to put here.
 . Click btn:[Create] to create the profile.
 
-
-
```

Comment:
Three blank lines before headings is standard, but we can fix this later on.

Reply:
I'm not sure I understand this one

Reply:
will fix it

Reply:
All of them: https://github.com/uyuni-project/uyuni-docs/wiki/markup-syntax#whitespace